examples/otel-logs-with-sourcetypes-splunk/README.md

```diff
@@ -0,0 +1,18 @@
+# Splunk HEC Example
+
+This example showcases how the collector can collect data from files and send it to Splunk Enterprise, associating with each source a different sourcetype.
```

Comment:
Would it be worthwhile to touch on or link to what 'sourcetype' means or do we expect users to already know if they're reading this?

Reply:
done